pkg/build/node/const.go

```diff
@@ -153,7 +154,7 @@ data:
             "ranges": [
               [
                 {
-                  "subnet": "{{ .PodCIDR }}"
+                  "subnet": "{{"{{ .PodCIDR }}"}}"
```

Comment:
this is kludgey but necessary, this template executes later on and is inside the manifest template :^)